openimaj/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>imaging</artifactId>
+        <groupId>science.aist.imaging</groupId>
+        <version>1.0.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>openimaj</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>science.aist.imaging</groupId>
+            <artifactId>api</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openimaj</groupId>
+            <artifactId>core-image</artifactId>
+            <version>1.3.10</version>
```

Comment:
I would move this version to the root pom.

Reply:
Fixed in [739c19f](https://github.com/FHOOEAIST/imaging/pull/14/commits/739c19fa2d18dad40ebac4676254c36ff07279f0)